_posts/2000-01-01-190-anonym-im-netz.md

```diff
@@ -37,7 +37,7 @@ Quelle: [Edward Snowden auf Twitter](https://twitter.com/Snowden/status/65358772
 
 Die einfachste Methode um Tor zu benutzen ist der [Tor Browser](https://www.torproject.org/). Es gibt ihn so wohl für PCs, Android als auch für [iOS](https://apps.apple.com/de/app/onion-browser/id519296448). Dieser ist ein angepasster Firefox, der jedoch den kompletten Verkehr durch das Tor Netzwerk leitet. Zusätzlich speichert der Browser selbst keine Daten dauerhaft auf deinem System. Die Verwendung des Tor Browser kann deinen Anonymität natürlich nicht gewährleisten, wenn Du auf den angesurften Seiten persönliche Daten preisgibst oder dich in rückverfolgbare Konten einloggst.
 
-Grundsätzlich solltest du bedenken, dass mögliche Überwacher Deines Anschlusses sehen können dass du Tor benutzt. Allerdings können sie nicht sehen was du machst, denn die Verbindung ins Netzwerk ist verschlüsselt. Die Nutzung von Tor an sich macht dich nicht verdächtig, denn außer dir machen das alleine [in Deutschland über 150.000 Meschen täglich](https://metrics.torproject.org/userstats-relay-country.html?country=de&events=off).
+Grundsätzlich solltest du bedenken, dass mögliche Überwacher Deines Anschlusses sehen können dass du Tor benutzt. Ermittler bewerten verschlüsselten Datenverkehr, der bei Tor oder mit VPNs entsteht, in bösartiger Absicht gerne auch als "klandestines Verhalten" um weiteres "Futter" für ihre Ermittlungskonstruktionen zu haben. Allerdings, und das ist das wichtigste, können sie nicht sehen was du machst, denn die Verbindung ins Netzwerk ist verschlüsselt. Die Nutzung von Tor an sich macht dich nicht verdächtig, denn außer dir machen das alleine [in Deutschland über 150.000 Meschen täglich](https://metrics.torproject.org/userstats-relay-country.html?country=de&events=off).
```

Comment:
Welchen Mehrwert siehst Du in der Ergänzung? 
Auf mich wirkt die Ergänzung abschreckend.

Reply:
Ich sehe darin einen informativen Mehrwert. Du hast recht das es Personen abschrecken kann, aber es wurde/wird eben leider auch für Ermittlungskonstruktionen so genutzt und fließt als Argument in weitere Anträge und Beschlüsse für Überwachungsmaßnahmen ein.

Reply:
Okay, ich lasse es mal so. :+1: